rpc/Cargo.toml

```diff
@@ -29,4 +29,8 @@ storage = { path = "../storage" }
 tezos_api = { path = "../tezos/api" }
 tezos_context = { path = "../tezos/context" }
 tezos_encoding = { path = "../tezos/encoding" }
-tezos_messages = { path = "../tezos/messages" }
\ No newline at end of file
+tezos_messages = { path = "../tezos/messages" }
+
+[dev-dependencies]
+assert-json-diff = "1.0.0"
+reqwest = { version = "0.10.0-alpha.2", features = ["blocking"] }
```

Comment:
Seems that thsi crate is not compatible with the changes introduced after migration to tokio 0.2